### ISSUE TITLE
BDD-V

### ISSUE THREAD

**Issue by Serge-weihao** [yzou2/cbst#9]
Could you provide the link of BDD-V dataset as your discription in the paper ?

**Comment by yzou2** [comment]
https://bdd-data.berkeley.edu/login.html

**Comment by yzou2** [comment]
https://bdd-data.berkeley.edu/

**Comment by Serge-weihao** [comment]
> 
> 
> https://bdd-data.berkeley.edu/

I have downed load bdd100k_seg.zip from the link and found 7000 images in training set and 1000 images in val, which is different from your description.

**Comment by Serge-weihao** [comment]
@yzou2 


    https://bdd-data.berkeley.edu/

I have downed load bdd100k_seg.zip from the link and found 7000 images in training set and 1000 images in val, which is different from your description.

**Comment by yzou2** [comment]
We used the data of visda2017 challenge in the paper. 

https://github.com/VisionLearningGroup/taskcv-2017-public/blob/master/segmentation/README.md

For evaluation the test set, you need to upload results to the server.
